package.json

```diff
@@ -29,7 +29,7 @@
     "install": "node-pre-gyp install --fallback-to-build"
   },
   "dependencies": {
-    "nan": "2.6.2",
+    "node-addon-api": "git+https://github.com/nodejs/node-addon-api.git",
```

Comment:
Do not merge this. This is a dependency on a floating master branch and will lead to breaking changes. Please pick a specific version to depend on.

Reply:
Yes you're right I want only understand where was the real problem if on the code or on ```node-addon-api`` version